src/Microsoft.AspNet.OData.Shared/EdmChangedObjectCollection.cs

```diff
@@ -57,5 +69,288 @@ private void Initialize(IEdmEntityType entityType)
             _edmType = new EdmDeltaCollectionType(new EdmEntityTypeReference(_entityType, isNullable: true));
             _edmTypeReference = new EdmCollectionTypeReference(_edmType);
         }
+
+        /// <summary>
+        /// Patch for Types without underlying CLR types
+        /// </summary>
+        /// <param name="originalCollection"></param>
+        /// <returns>ChangedObjectCollection response</returns>
+        public EdmChangedObjectCollection Patch(ICollection<EdmStructuredObject> originalCollection)
```

Comment:
I see you use this when patching a nested collection, if you can't get a patch handler for the nested collection. So one option, if we don't think it will be common to expose in the public API, is to make it internal for now.

---
In reply to: [615993621](https://github.com/OData/WebApi/pull/2455#discussion_r615993621) [](ancestors = 615993621)